api/index.js

```diff
@@ -36,6 +36,7 @@ export default async (req, res) => {
     disable_animations,
     border_radius,
     border_color,
+    role,
```

Comment:
well I totally forgot what this ownerAffiliation does, but why does this need to be controlled by the user?